ChattoAdditions/Source/Input/Media/Camera/MediaPicker.swift

```diff
@@ -24,19 +24,15 @@
 
 import UIKit
 
-public protocol ImagePickerDelegate: class {
-    func imagePickerDidFinish(_ picker: ImagePicker, mediaInfo: [UIImagePickerController.InfoKey: Any])
-    func imagePickerDidCancel(_ picker: ImagePicker)
+public protocol MediaPickerDelegate: class {
+    func imagePickerDidFinish(_ picker: MediaPicker, mediaInfo: [UIImagePickerController.InfoKey: Any])
+    func imagePickerDidCancel(_ picker: MediaPicker)
 }
 
-public protocol ImagePicker: class {
+public protocol MediaPicker: class {
     var controller: UIViewController { get }
 }
 
-public protocol ImagePickerFactory: class {
-    func makeImagePicker(delegate: ImagePickerDelegate) -> ImagePicker?
-}
-
-public struct ImagePickerStore {
```

Comment:
AFAIK this static variable was used to set default image picker from the application level (for functional tests, for example)

Reply:
I've removed this as it's a very bad way of injecting stuff, if this feature will be required some proper refactoring will be done